### PR TITLE
8 frontend create mock data

### PR DESCRIPTION
**Pull request by AndreaDellaValle**

### CODE REVIEW

**Review by Al-Pragliola — COMMENTED** [review]
I would also add the schema generation and verify in the Makefile, .drone.yml

File: web-client/src/ConfigurationContext.tsx [hunk 1]
```diff
@@ -25,7 +25,14 @@ export const ConfigurationProvider = ({ children }: ConfigurationProviderProps)
   useEffect(() => {
     if (env === 'development_local') {
       setConfiguration({
-        apiUrl: 'http://localhost:8080/api/v1',
+        apiUrl: 'http://localhost:8080/api/v1', // TILT URL
```
Comment (Al-Pragliola) [inline web-client/src/ConfigurationContext.tsx:28]: This is the local backend URL, maybe we can take also this address from environment variables

File: web-client/src/__tests__/App.test.tsx [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
-  render(<App />);
+  render(
+    <ConfigurationProvider>
+      <App />
+    </ConfigurationProvider>
+  );
   // ACT
   const textEl = await screen.findByTestId('welcome-text')
-  const fetchEl = await screen.findByTestId('fetch-text')
+  // const fetchEl = await screen.findByTestId('fetch-text')
```
Comment (Al-Pragliola) [inline web-client/src/__tests__/App.test.tsx:14]: let's remove commented code

File: web-client/sample-schema.yml [hunk 3]
```diff
@@ -0,0 +1,250 @@
+openapi: "3.0.0"
```
Comment (Al-Pragliola) [inline web-client/sample-schema.yml:1]: is this still needed?

File: web-client/README.md [hunk 4]
```diff
@@ -6,8 +6,20 @@ Desc: tbd
 
 ## Development
 
+### Local development with TILT
```
Comment (Al-Pragliola) [inline web-client/README.md:9]: mm not really with TILT

**Review by Al-Pragliola — CHANGES_REQUESTED** [review]

**Review by omissis — APPROVED** [review]
SBAM

**Review by Al-Pragliola — APPROVED** [review]
LGTM

**Pull request merged**